doc/source/data/index.rst

```diff
@@ -45,3 +45,19 @@ util module:
    :maxdepth: 1
 
    util
+
+SunPy and AstroPy Integration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+HelioPy is built to be used alongside SunPy and AstroPy. Before v0.6, HelioPy returned
+a Pandas DataFrame object. After implementation of physical units into the data, HelioPy
+returns a SunPy TimeSeries object. The TimeSeries object is capable of storing data like Pandas DataFrame,
+with added benefit of being used alongside AstroPy units.
+
+SunPy Documentation is available at http://docs.sunpy.org/
+You can also read more about `TimeSeries <http://docs.sunpy.org/en/stable/guide/data_types/timeseries.html>`_
+
+AstroPy Documentation is available at http://docs.astropy.org/
+You can also read more about `TimeSeries <http://docs.astropy.org/en/stable/units/>`_
+
+An example on how to use TimeSeries Data and AstroPy Units is also available in the Examples section.
```

Comment:
Could you add a link to the examples section? I'm not sure how to, but the spinx-gallery docs should help: https://sphinx-gallery.github.io/

Reply:
Got it! Thanks a lot for the reference!